vmdb/app/models/service_template_orchestration.rb

```diff
@@ -4,7 +4,8 @@ def orchestration_template
   end
 
   def orchestration_template=(template)
-    orchestration_templates.clear << template
+    orchestration_templates.clear
+    orchestration_templates << template if template
```

Comment:
This is not used by refresh. It is called once typically when adding a new catalog item for orchestration provisioning.